2025-02-05 07:35:47,769 - app.api.v1.endpoints.auth - INFO - Login attempt for user: bytepass@test.com
2025-02-05 07:35:47,924 - sqlalchemy.engine.Engine - INFO - select pg_catalog.version()
2025-02-05 07:35:47,925 - sqlalchemy.engine.Engine - INFO - [raw sql] ()
2025-02-05 07:35:47,932 - sqlalchemy.engine.Engine - INFO - select current_schema()
2025-02-05 07:35:47,933 - sqlalchemy.engine.Engine - INFO - [raw sql] ()
2025-02-05 07:35:47,937 - sqlalchemy.engine.Engine - INFO - show standard_conforming_strings
2025-02-05 07:35:47,938 - sqlalchemy.engine.Engine - INFO - [raw sql] ()
2025-02-05 07:35:47,941 - sqlalchemy.engine.Engine - INFO - BEGIN (implicit)
2025-02-05 07:35:47,947 - sqlalchemy.engine.Engine - INFO - SELECT users.email, users.full_name, users.hashed_password, users.is_active, users.is_superuser, users.id, users.created_at, users.updated_at 
FROM users 
WHERE users.email = $1::VARCHAR
2025-02-05 07:35:47,949 - sqlalchemy.engine.Engine - INFO - [generated in 0.00204s] ('bytepass@test.com',)
2025-02-05 07:35:48,180 - app.api.v1.endpoints.auth - INFO - User authenticated successfully: bytepass@test.com
2025-02-05 07:35:48,205 - app.api.v1.endpoints.auth - INFO - Generated tokens for user: bytepass@test.com
2025-02-05 07:35:48,206 - sqlalchemy.engine.Engine - INFO - ROLLBACK
2025-02-05 07:37:31,466 - app.api.v1.endpoints.auth - INFO - Token validation attempt
2025-02-05 07:37:31,468 - app.api.v1.endpoints.auth - ERROR - Token validation failed. Error: Invalid access token
2025-02-05 07:37:36,745 - app.api.v1.endpoints.auth - INFO - Token refresh attempt
2025-02-05 07:37:36,746 - sqlalchemy.engine.Engine - INFO - BEGIN (implicit)
2025-02-05 07:37:36,747 - sqlalchemy.engine.Engine - INFO - SELECT users.email, users.full_name, users.hashed_password, users.is_active, users.is_superuser, users.id, users.created_at, users.updated_at 
FROM users 
WHERE users.email = $1::VARCHAR
2025-02-05 07:37:36,749 - sqlalchemy.engine.Engine - INFO - [cached since 108.8s ago] ('bytepass@test.com',)
2025-02-05 07:37:36,752 - app.api.v1.endpoints.auth - INFO - Token refresh successful
2025-02-05 07:37:36,753 - sqlalchemy.engine.Engine - INFO - ROLLBACK
2025-02-05 07:37:46,896 - app.api.v1.endpoints.auth - INFO - Token validation attempt
2025-02-05 07:37:46,898 - sqlalchemy.engine.Engine - INFO - BEGIN (implicit)
2025-02-05 07:37:46,899 - sqlalchemy.engine.Engine - INFO - SELECT users.email, users.full_name, users.hashed_password, users.is_active, users.is_superuser, users.id, users.created_at, users.updated_at 
FROM users 
WHERE users.email = $1::VARCHAR
2025-02-05 07:37:46,900 - sqlalchemy.engine.Engine - INFO - [cached since 119s ago] ('bytepass@test.com',)
2025-02-05 07:37:46,904 - app.api.v1.endpoints.auth - INFO - Token validated successfully for user: bytepass@test.com
2025-02-05 07:37:46,905 - sqlalchemy.engine.Engine - INFO - ROLLBACK
